packages/cactus-plugin-ledger-connector-quorum/src/test/typescript/integration/plugin-ledger-connector-quorum/deploy-contract/v21.4.1-invoke-contract.test.ts

```diff
@@ -24,182 +24,196 @@ import { PluginRegistry } from "@hyperledger/cactus-core";
 
 const logLevel: LogLevelDesc = "INFO";
 const contractName = "HelloWorld";
+const testCase = "Quorum Ledger Connector Plugin";
 
-test("Quorum Ledger Connector Plugin", async (t: Test) => {
-  const containerImageVersion = "2021-05-03-quorum-v21.4.1";
-  const ledgerOptions = { containerImageVersion };
-  const ledger = new QuorumTestLedger(ledgerOptions);
-  test.onFinish(async () => {
+describe(testCase, () => {
+  // test("Quorum Ledger Connector Plugin", async (t: Test) => {
```

Comment:
@Leeyoungone No commented code unless you need it for something specific please :-)